keyboards/satan/keymaps/default/keymap.c

```diff
@@ -1,6 +1,8 @@
 #include QMK_KEYBOARD_H
 
-
+enum custom_keycodes {
+  SFT_ESC = SAFE_RANGE
+};
 
 // Used for SHIFT_ESC
 #define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```

Comment:
```suggestion
#define MODS_SHIFT_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```
Fix misleading name.